browser-test/src/support/applicant_questions.ts

```diff
@@ -343,11 +343,21 @@ export class ApplicantQuestions {
     await waitForPageJsLoad(this.page)
   }
 
+  async clickConfirmAddress() {
+    await this.page.click('text="Confirm address"')
```

Comment:
```suggestion
    await this.page.getByRole('button', { name: 'Confirm address' }).click()
```

(blocking) The `page.click` method is discouraged/deprecated. Playwright considers [using locators to be the best practice](https://playwright.dev/docs/best-practices#use-locators). Prefer using one of the `page.getXXXX` methods and only fallback to `page.locator` as a last resort.